Jointify/Model/PoseMachineLearning/PoseNet.swift

```diff
@@ -59,12 +59,17 @@ class PoseNet {
         static let jointColor: UIColor = UIColor.systemPink
         // Degrees that will be subtraced from the measured angle
         static let neutralNullAngle: Float = 90.0
+        // Minimum confidence that each joint has to conform to
+        static let confidenceThreshold: Double = 0.70
     }
     
     let side: Side
     private let jointSegments: [JointSegment]
+    private let selectedJointNames: [JointName]
     var degree: Float = 0.0
-    
+    // Dictionary for the coordinates of all joints
+    var jointPositions: [String: Float] = [:]
```

Comment:
Should this not be a `[Joint: Float]`?

Reply:
No, because I want a dictionary with one entry for each X and Y coordinate per joint. This is why I'm using Strings here.

Reply:
Again, access via a string is careless and error-prone. Imagine the `description` parameter of the enum you reference to im the String you unwrap in the dict access changes. No one knows that this is sensitive. 

You should use a `[Joint:CGPoint]` dict as that’s what you want to map: Joint to its coordinates. The `CGPoint` class is exactly integrated for this.